### DISCUSSION TITLE
How to add the Nextflow logo?

### DISCUSSION THREAD

**Discussion by mribeirodantas** [badges/shields#9583] [Q&A] [answered]
I would like the [Nextflow](https://github.com/nextflow-io/nextflow) logo on a badge, as we have for other technologies, such as npm, e.g.,![](https://img.shields.io/npm/v/npm.svg?logo=nodedotjs)

How can I help with this?

**Comment by calebcartwright** [comment]
Our docs on how to use/apply a logo can be found at https://shields.io/docs/logos

I don't see Nextflow in the Simple Icons set on https://simpleicons.org, so in the immediate term your only option would be to use our [Custom Logo feature](https://shields.io/docs/logos#custom-logos). You may also want to consider engaging with the Simple Icons team to see whether they'd add the Nextflow logo.

Please also note that although those docs mention Shields logos (a handful of our own custom logos), we've decided to finally drop those and will not be adding any new ones (#9476). This means that if Nextflow doesn't meet the Simple Icons criteria for inclusion then your only option will be the aforementioned Custom Logo route.

**Reply by mribeirodantas** [reply]
Thanks, @calebcartwright !

**Reply by IdoBar** [reply]
You can use this example [README.md](https://gist.github.com/IdoBar/423066b95616a8a27835216e918a1bcc) I made

**Comment by LitoMore** [comment, marked as answer]
The Nextflow icon is on Simple Icons now.

You can try this:

```markdown
![](https://img.shields.io/badge/Nextflow-0DC09D?logo=nextflow&logoColor=fff&style=flat-square)
```

![](https://img.shields.io/badge/Nextflow-0DC09D?logo=nextflow&logoColor=fff&style=flat-square)
